QualityControl/public/common/object/objectInfoCard.js

```diff
@@ -58,3 +117,32 @@ const infoPretty = (key, value) => {
   }
   return h('', value);
 };
+
+/**
+ * Default function to configure the info row vnode attributes
+ * @param {Notification} notification - Notification API
+ * @returns {function(string, string): object} object containing the constructed vnode attributes
+ */
+export const defaultRowAttributes = (notification) =>
+  (key, value) => ({
+    onclick: async (e) => {
+      // to allowing the default behaviour for clicking multiple times
+      const clickCount = e.detail;
+      if (clickCount === 1) {
+        if (!isContextSecure()) {
+          return;
+        }
+
+        notification.show('Value has been successfully copied to clipboard', 'success', 1500);
```

Comment:
The process of copying the value happens only on L141.
Should the notification not be placed after that line in the try catch?